docs/guide/java_object_graph_guide.md

```diff
@@ -93,6 +93,32 @@ public class Example {
 }
 ```
 
```

Comment:
Could we put this under advanced section and make it as third section title?

Reply:
Done.